picasso/chain.json

```diff
@@ -1,13 +1,13 @@
 {
   "$schema": "../chain.schema.json",
-  "chain_name": "composable",
+  "chain_name": "picasso",
   "chain_id": "centauri-1",
-  "website": "https://www.composable.finance/",
-  "pretty_name": "Composable",
+  "website": "https://www.picasso.xyz/",
+  "pretty_name": "Picasso",
   "status": "live",
   "network_type": "mainnet",
-  "bech32_prefix": "centauri",
-  "daemon_name": "centaurid",
+  "bech32_prefix": "pica",
```

Comment:
This prefix should be registered
https://github.com/satoshilabs/slips/blob/master/slip-0173.md